bolt/runtime/instr.cpp

```diff
@@ -272,10 +274,14 @@ class SimpleHashTable {
   /// avoid storing a pointer to it as part of this table (remember there is one
   /// hash for each indirect call site, so we wan't to minimize our footprint).
   MapEntry &get(uint64_t Key, BumpPtrAllocator &Alloc) {
+    if (!__bolt_instr_conservative) {
```

Comment:
Were you able to successfully build this? It's weird, but my compiler (clang 12) is complaining in this line with:

error: use of undeclared identifier '__bolt_instr_conservative'
    if (!__bolt_instr_conservative) {